parachains/pallets/collective-content/src/mock.rs

```diff
@@ -0,0 +1,95 @@
+// Copyright (C) 2023 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Test utilities.
+
+pub use crate as pallet_collective_content;
+use frame_support::{
+	ord_parameter_types,
+	traits::{ConstU32, ConstU64},
+};
+use frame_system::EnsureSignedBy;
+use sp_runtime::traits::IdentityLookup;
+
+frame_support::construct_runtime!(
+	pub enum Test where
+		Block = Block,
+		NodeBlock = Block,
+		UncheckedExtrinsic = UncheckedExtrinsic,
+	{
+		System: frame_system,
+		CollectiveContent: pallet_collective_content,
+	}
+);
+
+type BlockNumber = u64;
+type AccountId = u64;
+
+type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
+type Block = frame_system::mocking::MockBlock<Test>;
+
+ord_parameter_types! {
+	pub const CharterManager: u64 = 1;
```

Comment:
Same here, wondering if it's not better to have enum/string instead o `u64` to have better visibility

Reply:
Here different case, this is just a random account id we used with `EnsureSignedBy`